src/layouts/Connect.tsx

```diff
@@ -133,54 +142,45 @@ export const Connect: FC<MenuItemProps> = ({
-        <Tooltip>
-          <TooltipTrigger
-            className={cn('focus-visible:outline-none', customButtonStyle, customButtonWrapperStyle)}
-          >
-            <DropdownMenuTrigger asChild className={'focus-visible:outline-none'}>
-              <Button
-                colorScheme={!connected ? 'purple' : 'primaryGradient'}
-                size={breakpoint.isMobile || breakpoint.isTablet ? 'default' : 'sm'}
-                disabled={!canConnect}
-                className={cn(
-                  `min-w-[129px] min-md:min-w-[143px] px-1 py-1.75 focus-visible:outline-none 
+        <DropdownMenuTrigger asChild className={'focus-visible:outline-none'}>
+          <Button
+            colorScheme={!connected ? 'purple' : 'primaryGradient'}
+            size={breakpoint.isMobile || breakpoint.isTablet ? 'default' : 'sm'}
+            className={cn(
+              `min-w-[129px] min-md:min-w-[143px] px-1 py-1.75 focus-visible:outline-none 
                   flex justify-center`,
-                  customButtonStyle
-                )}
-                onClick={() => !connected && handleConnect()}
-              >
-                {connected && (
-                  <div
-                    className={`flex items-center justify-center border-4 dark:border-black-1 border-solid
+              customButtonStyle
+            )}
+            onClick={() => !connected && handleConnect()}
+          >
+            {connected && (
+              <div
+                className={`flex items-center justify-center border-4 dark:border-black-1 border-solid
                   border-grey-5 rounded-circle bg-grey-5 dark:bg-black-1 w-[24px] h-[24px] mr-1 overflow-hidden`}
-                  >
-                    <img
-                      className={'w-auto rounded-lg'}
-                      src={wallet?.adapter?.icon}
-                      alt={`${wallet?.adapter?.name}_icon`}
-                    />
-                  </div>
-                )}
-                {connectLabel}
-                {connected && (
-                  <img
-                    style={{
-                      transform: `rotate(${isOpen ? '0deg' : '180deg'})`,
-                      transition: 'transform 0.2s ease-in-out'
-                    }}
-                    className={`ml-1`}
-                    src={`/img/mainnav/connect-chevron.svg`}
-                    alt={'connect-chevron'}
-                  />
-                )}
-              </Button>
-            </DropdownMenuTrigger>
-          </TooltipTrigger>
-          <TooltipContent hidden={canConnect}>
-            We are sorry, {pathname.includes('trade') ? 'Trade' : 'Farm'} is currently unavailable in your location
-          </TooltipContent>
-        </Tooltip>
+              >
+                <img
+                  className={'w-auto rounded-lg'}
+                  src={wallet?.adapter?.icon}
+                  alt={`${wallet?.adapter?.name}_icon`}
```

Comment:
Here you do optional chaining 2 times, it transpiles to long checks. You can do all checks in separate variable through destructuring